src/components/AccessibilityDocumentsList/index.tsx

```diff
@@ -33,18 +33,17 @@ type DocumentsListProps = {
   documents: Document[];
   requestName: string;
   refetchRequest: () => any;
+  setConfirmationText: (text: string) => void;
```

Comment:
Could we access the hook directly here, instead of taking it in as a prop?

Reply:
I agree. I added this initially as a stopgap before fully embracing hooks/context. Since we're at that point now, I think it'd be best to avoid this.

On a separate note, I think a lot of the logic should get pulled out of `AccessibilityDocumentsList`. For example, `refetchRequest` and the mutation should probably should get taken out.

Reply:
I agree with both of these points. Thoughts on continuing in this PR vs. doing some of that in a follow-up?